flask/app/routes.py

```diff
@@ -431,28 +408,8 @@ def bulk_update():
             app.logger.debug(
                 "\tFamily ID '%s' already exists and won't be created.", family_id
             )
+        row["family_id"] = family_id
```

Comment:
Do we need to keep adding these in to make sure subsequent schemas in the hierarchy validate?

Reply:
As defined in `models.py`, `family_id` is a non-nullable field for the model `Participant`. When the schema is auto-generated for `Participant`, `family_id` thus becomes a `required` field. That's why I added `row["family_id"] = family_id` in order for input validation to be performed for  `Participant`. 

The same goes for the other insertions into the `row` dictionary.